nb/documentation/interactions_between_geometry/interactions_between_geometry.nb

```diff
@@ -82,6 +97,15 @@ Performs the hull operation on the input shapes.`;
 
 Hull(Arc(10), Box(5).move(12, 12, 0), Arc(20).x(40)).view();
 
+md`
+---
+### Mask
+Mask is used to create a void around something. Mostly used for adding play.
+`;
+
+const shaft = Arc(10, 10, 40).color('grey');
+shaft.mask(shaft.grow(1)).view();
```

Comment:
How about using the simpler form shaft.mask(grow(1)).view() ?

Reply:
Absolutely! I haven't quite gotten used to that yet, but I really should. It's an excellent addition.